components/sst_importer/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 TiKV Project Authors. Licensed under Apache-2.0.
+
+//! Importing RocksDB SST files into TiKV
+
+#[macro_use]
+extern crate lazy_static;
+#[macro_use]
+extern crate quick_error;
+#[macro_use]
+extern crate serde_derive;
+#[macro_use(slog_error, slog_warn, slog_info)]
+extern crate slog;
+#[macro_use]
+extern crate slog_global;
+
```

Comment:
Do we use to `extern crate tikv_alloc` explicitly in order to enable custom mallocator?

Reply:
We do have `extern crate tikv_alloc` in many libraries. I don't know if it's necessary or not, but I'll add it here.

Reply:
I have done so.